docs/assets/fhir-resource-ingestion-digestion-diagrams/bfd-pipeline-rda-ingestion.plantuml

```diff
@@ -0,0 +1,30 @@
+@startuml
+title BFD-Pipeline-RDA-GRPC Ingestion of Partially Adjudicated Claims Data Sequence
+participant "rda-pipeline-source"
+participant "rda-streaming-api"
+database "rda-db"
+participant "rda-pipeline-sink"
+database "bfd-db"
+note over "rda-streaming-api": Runs every 5 min in prod \n 1 hr in test
+"rda-pipeline-source" -> "rda-streaming-api": Smoke test to determine connectivity
+"rda-pipeline-source" -> "rda-db": Smoke test to determine connectivity
+"rda-streaming-api" --> "rda-pipeline-source"  
+"rda-db" --> "rda-pipeline-source"
+"rda-pipeline-source" -> "rda-db": Dead Letter Queue (DLQ) Processing
```

Comment:
Do we want to expand on what this processing entails?  It is similar to the regular processing but consists of these steps:

- read failed claims from DLQ
- process each failed claim from RDA API individually
- update database for each claim and mark as resolved or obsolete if possible

Reply:
Good question! The original intent may have been closer to something of a high level overview of the ingestion process, but the diagram already seems to be a little _too_ technical to remain at a high level. 

Let's try to get something closer to complete here.

Reply:
It's doable to add it in, and looks better, thx